pyproject.toml

```diff
@@ -17,7 +17,7 @@ classifiers = [
     'Topic :: Software Development :: Build Tools',
     "Operating System :: OS Independent",
 ]
-dependencies = ['requests==2.22.0', 'boto3==1.21.27','pandas==1.2.5', 'anndata==0.8.0', 'scanpy==1.9.1', 'numpy==1.21']
+dependencies = ['requests==2.22.0', 'boto3==1.21.27', 'pyflakes', 'pycodestyle', 'python-dotenv']
```

Comment:
we removed pandas and anndata and scanpy and numpy but they are needed for file-process, does it still work?